### PR TITLE
CLi v0.2 + Prettier

### PR DESCRIPTION
**Pull request by uF4No**
- Info for CLI new version (DEVRL-157)
- Fix Python menu (DEVRL-160)
- Fix quickstart deploy-script reference (DEVRL-161)
- Prettier

### CODE REVIEW

File: docs/tools/zksync-cli/README.md [hunk 1]
```diff
@@ -16,16 +16,22 @@ You can also run this via NPX with `npx zksync-cli@latest [COMMAND]`
...
-
-- `zksync-cli withdraw`: withdraws funds from zkSync Era to L1 (Goerli testnet). It will ask you to provide: the recipient wallet, the amount in ETH (eg 0.1) and the private key of the wallet you're sending the funds from.
+- `zksync-cli create [PROJECT_NAME]`: creates a new project in a new folder with the given project name. If the project name is not provided, it creates the project in the current folder, although this requires the folder to be empty.
+- `zksync-cli deposit`: deposits funds from L1 to zkSync Era. It will ask you to provide: the network (localnet, testnet, mainnet), the recipient wallet, the amount in ETH (eg 0.1) and the private key of the wallet you're sending the funds from.
+- `zksync-cli withdraw`: withdraws funds from zkSync Era to L1. It will ask you to provide: the network (localnet, testnet, mainnet), the recipient wallet, the amount in ETH (eg 0.1) and the private key of the wallet you're sending the funds from.
+- `zksync-cli help`: returns the version and information about the different commands.
```
Comment (dutterbutter) [inline docs/tools/zksync-cli/README.md:22]: Can add `confirm-withdraw` to list of commands

Comment (uF4No) [reply docs/tools/zksync-cli/README.md]: fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/633/commits/fcaf423dfe5f1be431438a843e107ce3f3244e6b

File: docs/README.md [hunk 2]
```diff
@@ -21,56 +21,52 @@ contributors: false
...
             <p>Differences between zkSync Era and Ethereum.</p>
          </div>
     </a>
-    <a href="./api/" class="card">
+    <a href="./reference/concepts/hyperscaling.html" class="card">
       <div class="content">
-        <h3>SDK References</h3>
-        <p>References for your favorite web3 SDKs.</p>
+        <h3>Hyperscaling</h3>
+        <p>Scaling Ethereum beyong Era with hyperchains.</p>
```
[the remark this answers is not in the document]
Comment (uF4No) [reply docs/README.md]: fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/633/commits/4df97c94bcb8d6395e13d5261c8c494e8e4f9c89

**Review by idea404 — APPROVED** [review]
Approving here

**Review by MexicanAce — APPROVED** [review]
LGTM

**Pull request merged**